### PR TITLE
 Fix error in css styles

### PR DESCRIPTION
**Pull request by maherramadan78**
I add the contact form to the pr.
I fix the design in the css.style file.

### CODE REVIEW

**Comment by maherramadan78** [comment]
I fix some errors in the CSS style.
I change the color of the button from gris to violet as it looks in the Figma design.
I add the contact form to the portfolio mobile version.

File: index.html [hunk 1]
```diff
@@ -222,5 +222,35 @@ <h2>Multi-Post Stories</h2>
...
+    <section class="contact-form">
+   
+      <h2 class="contact-us">Contact Us</h2>
+      <p class="paragraph3">If you have an application you are
+        interested in developing, a feature that you
+        need built or a project that needs coding.
+        I'd love to help with it </p>
+    
+      <form action="https://formspree.io/forms/meqdknwb/integration" method="post" class="form">
+      <input type="text" id="name" name="name" placeholder="User name" maxlength="30" required>
```
Comment (Kossi-stack) [inline index.html]: - [ ] Kindly replace the User name with a first name or last name

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @maherramadan78,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (maherramadan78) [reply index.html]: ok , i will , thank you sir.

**Review by levy002 — APPROVED** [review]
### Hello @maherramadan78 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**